
<!-- 필드에서 발생한 오류 표시 -->
<!-- validators 검증에 실패한 경우 표시됨 -->
{% for field, errors in form.errors.items() %}
<div class="alert alert-danger" role="alert">
    <strong>{{ form[field].label }}</strong>: {{ ', '.join(errors)}}
</div>
{% endfor %}

<!-- flash 거치면서 발생한 오류 -->
<!-- flash('이미 존재하는 사용자입니다')와 같은 로직으로 표시됨 -->
{% for message in get_flashed_messages() %}
<div class="alert alert-danger" role="alert">
    {{ message }}
</div>
{% endfor %}